view/theme/frio/css/style.css

```diff
@@ -814,6 +827,35 @@ nav.navbar a {
 }
 
 /* Dropdown Menus */
+.nav > li > .btn-link {
+    position: relative;
+    display: block;
+    padding: 10px 15px;
+}
+.nav > li > .btn-link:focus,
+.nav > li > .btn-link:hover {
+    text-decoration: none;
+    background-color: #eee;
+}
+.nav .open > .btn-link,
+.nav .open > .btn-link:focus,
+.nav .open > .btn-link:hover {
+    background-color: #eee;
+    border-color: #337ab7;
+}
+.nav-pills > li > .btn-link {
+    border-radius: 4px;
+}
+.dropdown-menu > li > :focus,
+.dropdown-menu > li > :hover {
```

Comment:
I can't find anything impacted by its removal, I assume it's a straight import from Bootstrap and converted to support buttons, and it's been since superseded by custom frio rules also brought to play nice with buttons. I'm removing it for now.